roles/addons/diskless/readme.rst

```diff
@@ -122,17 +122,20 @@ Select the kernel you want to use wit this image from the list of available kern
  
 Choose a standard image, and give a size.
 
-Notes:
+Proposal: 
 
-* The size recommended for the Standard or Custom is 5G
-* The size of the final image booted and loaded in RAM is approximately : 1.2GB for Standard image, 250MB for Small image, 130MB for Minimall image and 1.2GB for Custom image
+* The size recommended for the Standard or Custom profile is 5G.
+* The size of the final image booted and loaded in RAM is approximately :
+  - 1.2GB for Standard image
```

Comment:
a blank line is missing, [check the rendering](https://github.com/bluebanquise/bluebanquise/blob/3484624dba92a7bc08fafc1da65ee93a37f5e48c/roles/addons/diskless/readme.rst)

Reply:
fixed